### PR TITLE
[14.0] [MIG] website_sale_stock_list_preview: Migration to 14.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
@Tecnativa 

please @sergio-teruel @Tardo review this

### CODE REVIEW

File: website_sale_stock_list_preview/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module extends the functionality of e-Commerce to give
+the buyer the information about the stock on the preview of the products.
```
Comment (LudLaf) [inline website_sale_stock_list_preview/readme/DESCRIPTION.rst]:
```suggestion
This module extends e-Commerce's functionality to give the buyer information about what's in stock when previewing the products
```

File: website_sale_stock_list_preview/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+* Edit some products on the option Availability of e-commerce
```
Comment (LudLaf) [inline website_sale_stock_list_preview/readme/USAGE.rst]:
```suggestion
* Edit the 'Availability' option under 'eCommerce' tab in your selected product.
```

File: website_sale_stock_list_preview/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+* Go to the shop on the website and active Add to Cart view on Customize menu.
```
Comment (LudLaf) [inline website_sale_stock_list_preview/readme/USAGE.rst]:
```suggestion
* Go to the website's shop and activate the 'Add to Cart' view under 'Customize' menu.
```

File: website_sale_stock_list_preview/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+* You can see on your products the results of the options that you had choose.
```
Comment (LudLaf) [inline website_sale_stock_list_preview/readme/USAGE.rst]:
```suggestion

You can see, on your products, the options you have chosen.
```

**Review by LudLaf — CHANGES_REQUESTED** [review]
Take a look at the README

**Comment by CarlosRoca13** [comment]
Changes done @LudLaf

**Review by pilarvargas-tecnativa — APPROVED** [review]
locally tested :+1:

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**